### PR TITLE
Update co2eq_parameters.js

### LINKED ISSUE #218: CO2 intensity for hydropower in Norway 10x too large
The CO2 intensity for hydropower used here seems to be based on the 2014 IPCC *median* number, which is 24 g/kWh. However, the actual intensity varies greatly from dam to dam, mostly due to local climate. It is highest in tropical climates due to greater methane production in reservoirs there, and is minimal in cold climates. So one would expect the numbers for Europe in general and Norway in particular to lie towards the lower part of the 1.0 g/kWh to 2200 g/kWh range the IPCC gives. This is backed up by [a study focusing on Norwegian hydropower](http://ostfoldforskning.no/media/1056/734-1.pdf), which finds 1.9 g/kWh for the power station they studied.

This makes the CO2 intensity numbers for Norway in your model 13 times too high, and since Norway uses almost exclusively hydropower, the total CO2 intensity numbers for electricity production for the country should also be more than 10 times too high.

It is probably safe to assume that hydropower in other European countries is also much closer to 2 g/kWh than the 24 g/kWh that's being assumed.

### PR DESCRIPTION
**Pull request by brunolajoie**
Update Norway Hydro carbon intensity.
fixes #218

### CODE REVIEW

**Comment by brunolajoie** [comment]
@oli i've updated the readme. what do you mean by update my branch?

File: README.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ The carbon intensity of each country is measured from the perspective of a consu
 The carbon intensity of each type of power plant takes into account emissions arising from the whole lifecyle of the plant (construction, fuel production, operational emissions, and decomissioning). GHG intensity factors used in the map are detailed in [co2eq-parameters.js](https://github.com/corradio/electricitymap/blob/master/shared/co2eq_parameters.js). These numbers come from the following scientific peer reviewed litterature:
```
Comment (corradio) [inline README.md]: Can you rename GHG intensity factors to carbon intensity factors?

File: README.md [hunk 2]
```diff
@@ -17,9 +17,11 @@ The carbon intensity of each country is measured from the perspective of a consu
 The carbon intensity of each type of power plant takes into account emissions arising from the whole lifecyle of the plant (construction, fuel production, operational emissions, and decomissioning). GHG intensity factors used in the map are detailed in [co2eq-parameters.js](https://github.com/corradio/electricitymap/blob/master/shared/co2eq_parameters.js). These numbers come from the following scientific peer reviewed litterature: 
 - IPCC 2014 Assessment Report is used as reference in most instances (see a summary in the [wikipedia entry](https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources))
 
-Country specific emissions:
+Country-specific powerplant carbon-intensity factors:
```
Comment (corradio) [inline README.md]: I would remove "powerplant", because it's more production type than power plant (which is spelled in two words btw)

**Comment by corradio** [comment]
I'm not sure there a dash (-) between carbon and intensity, but that's fine anyway. Thanks @brunolajoie !

**Pull request merged**